app/controllers/markets_controller.rb

```diff
@@ -0,0 +1,45 @@
+class MarketsController < ApplicationController
+
+  def new
+    @market = Market.new
+  end
+
+  def create
+    @market = Market.new(market_edit_params[:market])
+    if @market.save
+      redirect_to markets_path
+    else
+      render :new
+    end
+  end
+
+  def index
+    @markets = Market.all
+    @market = Market.first
+    @vendor = Vendor.first
+  end
+
```

Comment:
Yeah- I think we put those there after developing another view that needed
them, realized we didn't for that one, and didn't take them back out.

On Tue, May 3, 2016 at 8:07 PM, Dreedle <notifications@github.com> wrote:

> In app/controllers/markets_controller.rb
> <https://github.com/Ada-C5/FarMarRails/pull/9#discussion_r61986318>:
>
> > +
> > +  def create
> > +    @market = Market.new(market_edit_params[:market])
> > +    if @market.save
> > +      redirect_to markets_path
> > +    else
> > +      render :new
> > +    end
> > +  end
> > +
> > +  def index
> > +    @markets = Market.all
> > +    @market = Market.first
> > +    @vendor = Vendor.first
> > +  end
> > +
>
> I can't figure out what @market <https://github.com/market> and @vendor
> <https://github.com/vendor> are used for in the view. Maybe I'm just
> missing it- double check that you actually need this.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/Ada-C5/FarMarRails/pull/9/files/da2a02f087c82df712f5ab599b65813a6ffd05d7#r61986318>
>